pyproject.toml

```diff
@@ -25,11 +25,11 @@ classifiers = [
 python = "^3.7"
 async-timeout = "^4.0.1"
 typing-extensions = { version = "^4.2.0", python = "<3.8" }
-dbus-next = { version = "^0.2.2", markers = "platform_system=='Linux'" }
 pyobjc-core = { version = "^8.5", markers = "platform_system=='Darwin'" }
 pyobjc-framework-CoreBluetooth = { version = "^8.5", markers = "platform_system=='Darwin'" }
 pyobjc-framework-libdispatch = { version = "^8.5", markers = "platform_system=='Darwin'" }
 bleak-winrt = { version = "^1.1.1", markers = "platform_system=='Windows'" }
+dbus-fast = {version = ">=1.4.0", markers = "platform_system == \"Linux\""}
```

Comment:
Let's use caret notation for the version to avoid breaking changes. And we can use single quotes to avoid the escape characters.

Reply:
4432e85ed88d2eda01dc63a95f759ececf22b955